scratchpad/tornado/test_tornado_io.py

```diff
@@ -1,8 +1,12 @@
-from tornado_io import *
+# Copyright 2014 National Research Foundation (South African Radio Astronomy Observatory)
+# BSD license - see LICENSE for details
+
+from __future__ import absolute_import, division, print_function
 
 from tornado.util import ObjectDict
-from katcp import Sensor
 
+from katcp import Sensor
+from tornado_io import *
```

Comment:
implicit is better than explicit, perhaps instead of importing * rather import the module specifically.